manual/source/Tutorial/DataStructures/Graphs.rst

```diff
@@ -284,8 +286,8 @@ Be aware that the word external is a hint that the information is stored indepen
 Property maps are simply :dox:`String Strings` of a property type and are indexed via the already well-known vertex and edge descriptors.
 
 Lets see how we can define a vertex property map for the city names.
-Our property type is a :dox:`String` of a city name type, a char string.
-We only have to create and :dox:`Graph#resizeVertexMap resize` this map so that it can hold information on all vertices.
+Our property type is a :dox:`String` of a city name, more explicitly a char string (``TCityName``). The vertex property map should hold several names so we define a String of Strings (``TProperties``).
```

Comment:
Since I can see the source code immediately after this. mentioning of  (``TCityName``) or  (``TProperties``) is too much detail I think.